app/views/layouts/_logged_content.html.slim

```diff
@@ -1,22 +1,31 @@
 - if current_user.banned?
   div.banned.bg-danger Ваш обліковий запис заблоковано
-ul.navbar-nav.nav.pull-right
-  - if current_user.admin? || current_user.moderator?
-      li.nav-text-link
+- if current_user.admin? || current_user.moderator?
+  li.dropdown
+    a.dropdown-toggle[href="#" data-toggle="dropdown"]
+      | Меню модератора
```

Comment:
Moderators and admins also can send issues and watch their profile, so we need links for this in top menu not only for reporters but also for admins and moders